assets/package.json

```diff
@@ -5,6 +5,7 @@
   "repository": {},
   "license": "MIT",
   "scripts": {
+    "analyze-bundle": "BUNDLE_ANALYZER_ENABLED=true node --max-old-space-size=8192 node_modules/webpack/bin/webpack.js --mode production",
```

Comment:
If you run this, you'll get a browser window that you can explore what's in each .js chunk.